sig/datadog/tracing/contrib/hanami/plugin.rbs

```diff
@@ -0,0 +1,3 @@
+class Object
+  def inherited: (untyped base) -> untyped
+end
```

Comment:
Minor: This doesn't seem correct, looking at https://github.com/DataDog/dd-trace-rb/blob/master/lib/datadog/tracing/contrib/hanami/plugin.rb . I think the autogenerator got confused about the anonymous module being inserted into Hanami...